Projects/Smoljankin/Task_301/Task_301/Program.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+
+namespace Task_301
+{
+    class Person
+    {
+        string firstname, lastname, gender, tel;
+        int age;
+        
+        public string FirstName
+        {
+            set => firstname = value;
+            get => firstname;
+        }
+
+        public string LastName
+        {
+            set => lastname = value;
+            get => lastname;
+        }
+
+        public string Gender
+        {
+            set => gender = value;
+            get => gender;
+        }
+
+        public string Tel
+        {
+            set => tel = value;
+            get => tel;
+        }
+
+        public int Age
+        {
+            set => age = value;
+            get => age;
+        }
+
+        public Person()
+        {
+            firstname = "";
+            lastname = "";
+            gender = "";
+            tel = "";
+            age = 0;
+        }
+        public Person(string firstname, string lastname, int age, string gender, string tel)
+        {
+            this.firstname = firstname;
+            this.lastname = lastname;
+            this.gender = gender;
+            this.tel = tel;
+            this.age = age;
+        }
+        public void SetFirstName(string firstname)

```

Comment:
Do we need these methods? This logic is already implemented in the Fields
        public string FirstName
        {
            set => firstname = value;
            get => firstname;
        }